app/waf/src/models/monitor.model.ts

```diff
@@ -152,35 +224,62 @@ export class Monitor extends CommonEntity {
   @property({
     type: 'string',
     default: '',
+    openapi: {
+      minLength: '1',
+      maxLength: '100',
+    },
   })
   ciphers: string;
 
   @property({
     type: 'string',
+    openapi: {
+      minLength: '1',
+      maxLength: '100',
```

Comment:
It may be a longer string. Let's fix it when we need to support this setting.

Reply:
Change it to 102400 temporarily and we could change it further later when we need to supoort this.